pkgs/tools/misc/coreutils/default.nix

```diff
@@ -17,21 +17,14 @@ with lib;
 
 stdenv.mkDerivation (rec {
   pname = "coreutils";
-  version = "8.31";
+  version = "8.32";
 
   src = fetchurl {
     url = "mirror://gnu/${pname}/${pname}-${version}.tar.xz";
-    sha256 = "1zg9m79x1i2nifj4kb0waf9x3i5h6ydkypkjnbsb9rnwis8rqypz";
+    sha256 = "1yjcrh5hw70c0yn8zw55pd6j51dj90anpq8mmg649ps9g3gdhn24";
   };
 
-  patches = optional stdenv.hostPlatform.isCygwin ./coreutils-8.23-4.cygwin.patch
-         # Fix failing test with musl. See https://lists.gnu.org/r/coreutils/2019-05/msg00031.html
-         # To be removed in coreutils-8.32.
-         ++ optional stdenv.hostPlatform.isMusl ./avoid-false-positive-in-date-debug-test.patch
-         # Fix compilation in musl-cross environments. To be removed in coreutils-8.32.
-         ++ optional stdenv.hostPlatform.isMusl ./coreutils-8.31-musl-cross.patch
-         # Fix compilation in android-cross environments. To be removed in coreutils-8.32.
-         ++ [ ./coreutils-8.31-android-cross.patch ];
+  patches = optional stdenv.hostPlatform.isCygwin ./coreutils-8.23-4.cygwin.patch;
```

Comment:
Did you check that these don't apply anymore or just removed based on the comment?

Reply:
The latter

Reply:
The remaining patch doesn't apply
cc @tobim

Reply:
@tobim , sorry for the noise.
The patch came in at 221b5a5be5eb698c696b728b0a9b6646422cd9c7

It doesn't apply against 8.31, so I propose to remove cygwin-specific things.

Reply:
Done! Nice catch @veprbl.

Reply:
@lovesegfault Could you, please, remove the other bit too?
https://github.com/NixOS/nixpkgs/blob/51fa4f0d9a9d6491d9b39a0d1e34f804bd05587c/pkgs/tools/misc/coreutils/default.nix#L67
If the patch is gone, nothing will change configure.ac, so there is no need to regenerate.

Reply:
Done, removed the `musl` part too as that patch is now gone as well.